src/infer_stats.cc

```diff
@@ -111,8 +111,12 @@ InferenceStatsAggregator::UpdateSuccessWithDuration(
     metric_reporter->IncrementCounter(
         "compute_output_duration", compute_output_duration_ns / 1000);
     // Summary Latencies
-    metric_reporter->ObserveSummary(
-        "request_duration", request_duration_ns / 1000);
+    const auto& reporter_config = metric_reporter->Config();
+    // FIXME [DLIS-4762]: request summary is disabled when cache is enabled.
```

Comment:
Instead of silently skipping the request_duration in Summary, shouldn't we throw an error when parsing the CLI options? 
"summary metric config not supported when caching is enabled."
I guess we also depend upon whether model_config specifies caching setting, but for now I think it will be okay to be more conservative and throw this error.

Reply:
The request_duration summary is the only metric with an issue around caching. The queue, compute, and cache hit/miss summaries are all still correct and could be reported as summaries.

Since the metrics are not yet individually toggle-able, I only disabled the broken request_duration summary rather than all summary metrics. I figured this would still give some useful information to users that want this feature with cache-enabled models for now.